customers/schema/types.py

```diff
@@ -134,6 +136,7 @@ class Meta:
     )
     winter_storage_leases = DjangoConnectionField(WinterStorageLeaseNode)
     orders = DjangoConnectionField("payments.schema.OrderNode")
+    offers = DjangoFilterConnectionField("payments.schema.BerthSwitchOfferNode")
```

Comment:
Do we actually need the filter connection? 🤔

Reply:
Whoops, that slipped there :grin: